### DISCUSSION TITLE
swappy segfaults when using the colorchooser

### DISCUSSION THREAD

**Discussion by bbigras** [lovesegfault/nix-config#275] [General]
>> Btw, unstable now has swappy "A Wayland native snapshot editing tool, inspired by Snappy on macOS", it's pretty basic but I use those annotation tools, like flameshot when not using wayland, a lot.

> That's really cool! It needs a gtk wrapper though, right now clicking the colorchooser causes it to segfault :D

"segfault" haha :wink: .

Do you still have that problem? I don't on my laptop. I wonder if I have something special in my config.

**Comment by lovesegfault** [comment]
Yep, I still have it apparently:
```
❯ grim -g "$(slurp)" - | swappy -f -

(swappy:16229): GLib-GIO-ERROR **: 00:10:43.594: Settings schema 'org.gtk.Settings.ColorChooser' is not installed
zsh: done                      grim -g "$(slurp)" - |
zsh: trace trap (core dumped)  swappy -f -
```

And the full backtrace: https://gist.github.com/b1c5bc90acb17cee7261916a7a0edd5e


Overall I think it's a mix of somehow me missing that colorchooser schema, and some SW bug that causes it to coredump when a schema is missing.

Despite my name I do not, in fact, love segfaults :D

**Comment by lovesegfault** [comment]
For the record, this was eventually fixed and I now use swappy :)
